helpers/playerHelper.js

```diff
@@ -78,6 +87,7 @@ exports.parseCollection = function(players, subresources) {
       switch (resource) {
         case 'stats':
           player.stats = self.mapStats(p[idx + 1].player_stats);
+          player.points = self.mapPoints(p[idx + 1].player_points);
```

Comment:
yea... the tests... they're kind of an ongoing project...

I've gotten more into doing them in my real life job, but adding them late now is not the most exciting thing with the limited free time I have (terrible, right?)

I'll run through some stuff on this branch tonight and get back to you!